Networking/Networking/Model/ShippingLabel/Packages/WooShippingPackagesResponse.swift

```diff
@@ -0,0 +1,90 @@
+import Foundation
+import Codegen
+
+/// Represents store options, a list of saved Shipping Label Packages (custom and predefined) for the WooCommerce Shipping extension.
+///
+public struct WooShippingPackagesResponse: Equatable, GeneratedFakeable, GeneratedCopiable {
+
+    /// Store options
+    public let storeOptions: ShippingLabelStoreOptions
+
+    /// Saved custom packages
+    public let customPackages: [WooShippingCustomPackage]
+
+    /// Saved (activated) predefined options
+    public let savedPredefinedOptions: [WooShippingPredefinedSavedOption]
```

Comment:
Just a followup thought I had here: It might be worth parsing the response to set this property as `[WooShippingPredefinedOption]` instead of `[WooShippingPredefinedSavedOption]`. That way we can use the list of saved options directly in the UI without having to parse them at that point.

Reply:
Gotcha!
So basically to use all the options we get and store in `allPredefinedOptions`, and just use the ones whoose IDs match the saved ones?